versions/3.1.0.md

```diff
@@ -2235,7 +2235,7 @@ description: Pets operations
 
 #### <a name="referenceObject"></a>Reference Object
 
-A simple object to allow referencing other components in the OpenAPI document, internally and externally.
+A simple object to allow referencing other objects in the OpenAPI document, internally and externally. Targets of a reference do not need to be contained in a components section and for external references, targets MAY exist within any compatible resource. Referenced objects are subject to the same constraints as inline objects.
```

Comment:
Should we stick to a single term here? We use `targets` and `referenced objects` to mean the same thing.